configs/vitest.config.ts

```diff
@@ -46,12 +46,6 @@ export default defineConfig(({ mode }) => {
     },
     resolve: {
       conditions: testSSR ? ["node"] : ["browser", "development"],
-      // fix issues with stores and node@20:
-      ...(!testSSR && {
-        alias: {
-          "solid-js/store": "solid-js/store/dist/store.js",
```

Comment:
@atk I needed to remove it so that both `solid-js/store` and `solid-js/web` resolve to the dev runtime version. Instead of the `store` resolving to prod, and `web` to dev, causing `isDev` to be `true` while the dev features were missing.